src/AppBundle/DataFixtures/ORM/PostFixtures.php

```diff
@@ -21,10 +21,8 @@
 use Symfony\Component\DependencyInjection\ContainerAwareTrait;
 
 /**
- * Defines the sample data to load in the database when running the unit and
- * functional tests.
- *
- * Execute this command to load the data:
+ * Defines the sample blog posts to load in the database before running the unit
+ * and functional tests. Execute this command to load the data:
```

Comment:
Once we update the PHPCsFixer to 2.0, we can disable that rule because `:` should be valid as the ending of a phrase.